docs/source/tutorials/day_of_the_week.qmd

```diff
@@ -0,0 +1,314 @@
+        rt_rv = metaclass.TransformedRandomVariable(
+            name="Rt_rv",
+            base_rv=base_rv,
+            transforms=transformation.ExpTransform(),
+        )
+
+        return rt_rv(n_steps=n_steps, **kwargs)
+
+
+rtproc = MyRt(
+    metaclass.DistributionalRV(
+        name="Rt_random_walk_sd", dist=dist.HalfNormal(0.025)
+    )
+)
+```
+
+3. We defined the observation model:
+
+```{python}
+# | label: obs-model
+# | code-fold: true
+# we place a log-Normal prior on the concentration
+# parameter of the negative binomial.
+nb_conc_rv = metaclass.TransformedRandomVariable(
+    "concentration",
+    metaclass.DistributionalRV(
+        name="concentration_raw",
+        dist=dist.TruncatedNormal(loc=0, scale=1, low=0.01),
+    ),
+    transformation.PowerTransform(-2),
+)
+
+# now we define the observation process
+obs = observation.NegativeBinomialObservation(
+    "negbinom_rv",
+    concentration_rv=nb_conc_rv,
+)
+```
+
+4. And finally, we built the model:
+
+```{python}
+# | label: init-model
+hosp_model = model.HospitalAdmissionsModel(
+    latent_infections_rv=latent_inf,
+    latent_hosp_admissions_rv=latent_hosp,
+    I0_rv=I0,
+    gen_int_rv=gen_int,
+    Rt_process_rv=rtproc,
+    hosp_admission_obs_process_rv=obs,
+)
+```
+
+Here is what the model looks like without the day-of-the-week effect:
+
+```{python}
+# | label: fig-output-admissions-padding-and-weekday
+# | fig-cap: Hospital Admissions posterior distribution without weekday effect
+import jax
+import numpy as np
+
+# Model without weekday effect
+hosp_model.run(
+    num_samples=2000,
+    num_warmup=2000,
+    data_observed_hosp_admissions=daily_hosp_admits,
+    rng_key=jax.random.key(54),
+    mcmc_args=dict(progress_bar=False),
+)
+
+# Plotting the posterior
+out = hosp_model.plot_posterior(
+    var="latent_hospital_admissions",
+    ylab="Hospital Admissions",
+    obs_signal=np.pad(
+        daily_hosp_admits.astype(float),
+        (gen_int_array.size, 0),
+        constant_values=np.nan,
+    ),
+)
+```
+
+
+## Round 2: Incorporating day-of-the-week effects
+
+We will re-use the infection to admission interval and infection to hospitalization rate from the previous model. But we will also add a day-of-the-week effect. To do this, we will add two additional arguments to the latent hospital admissions random variable: `day_of_the_week_rv` (a `RandomVariable`) and `obs_data_first_day_of_the_week` (an `int` mapping days of the week from 0:6, zero being Monday). The `day_of_the_week_rv`'s sample method should return a vector of length seven; those values are then broadcasted to match the length of the dataset. Moreover, since the observed data may start in a weekday other than Monday, the `obs_data_first_day_of_the_week` argument is used to offset the day-of-the-week effect.
+
+For this example, the effect will be passed as a Dirichlet distribution with a concentration parameter of 1/7: [^note-other-examples]:
```

Comment:
Why concentration of 1/7? We should probably multiply the Dirichlet by 7, as in the WW model. You can do this with a `TransformedRandomVariable`:

```python
TransformedRandomVariable(
    "hosp_wday_effect",
    DistributionalRV(
        "hosp_wday_effect_raw", dist.Dirichlet(concentration=jnp.ones(7))
    ),
    transforms.AffineTransform(loc=0, scale=7),
)
```